services/core/MasterDriverAgent/master_driver/driver.py

```diff
@@ -378,12 +378,25 @@ def publish_cov_value(self, point_name, point_values):
             meta = {point_name: self.meta_data[point_name]}
             message = [results, meta]
 
+            depth_first_topic, breadth_first_topic = self.get_paths_for_point(
+                point_name)
+
             if self.publish_depth_first:
-                self._publish_wrapper(self.all_path_depth,
+                self._publish_wrapper(depth_first_topic,
                                       headers=headers,
                                       message=message)
             #
             if self.publish_breadth_first:
+                self._publish_wrapper(breadth_first_topic,
+                                      headers=headers,
+                                      message=message)
```

Comment:
The message is wrong for the publish_depth_first and publish_breadth_first publishes. The results are not supposed to be wrapped in a dictionary in those cases. See https://github.com/VOLTTRON/volttron/blob/44732ba303602bdb04658a7e2fe7b3f92877dc61/services/core/MasterDriverAgent/master_driver/driver.py#L272 for what is expected in a single value publish.